test/message_test.coffee

```diff
@@ -0,0 +1,41 @@
+# Assertions and Stubbing
+chai = require 'chai'
+sinon = require 'sinon'
+chai.use require 'sinon-chai'
+
+{ expect } = chai
+
+# Hubot classes
+Robot = require '../src/robot.coffee'
+{ CatchAllMessage, EnterMessage, Message, TextMessage } = require '../src/message'
+
+describe 'Message', ->
+  beforeEach ->
+    @robot = new Robot null, 'mock-adapter', yes, 'TestHubot'
+    @robot.run
+
+    @user = @robot.brain.userForId '1', {
+      name: 'hubottester'
+      room: '#mocha'
+    }
```

Comment:
Can we minimize the dependencies here and just instantiate a `User` directly? You shouldn't need a `Robot` at all.